.gitignore

```diff
@@ -1,2 +1,7 @@
+test/resources/local_mode_lock
+.tox/
+.coverage
+*.egg-info/
+__pycache__/
```

Comment:
Thanks for updating this! Consider adding the following to ignore the dist folder as well:

```
dist/
```